src/data/markdown/docs/02 javascript api/11 k6-execution/1-Execution.md

```diff
@@ -0,0 +1,137 @@
+
+export default function() {
+	console.log(exec.scenario.name) // myscenario 
+}
+```
+
+The module has a default export that has the following keys `scenario`, `instance` and `vu` which in themselves expose information specific to the context in which it's being accessed.
+
+### Scenario
+
+| Field               | Type    | Description                                                              |
+|---------------------|---------|--------------------------------------------------------------------------|
+| name                | string  | It is the assigned name of the running scenario.                                |
+| executor            | string  | It is the name of the running [Executor](https://k6.io/docs/using-k6/scenarios/#executors).                                                |
+| startTime           | integer | It represents the unix timestamp in milliseconds when the scenario started.                                                                           |
+| progress            | float   | Percentage in a 0 to 1 interval of the scenario progress.                    |
+| iterationInInstance | integer | The unique identifier of the current iteration across the current instance. |
+| iterationInTest     | integer | The unique identifier of the current iteration across the entire test. It is unique both in local, in cloud and in distributed or segmented executions. |
+
+### Instance
+
+| Field               | Type    | Description                                                              |
+|---------------------|---------|--------------------------------------------------------------------------|
+| iterationsInterrupted                | integer  | The amount of iterations interrupted.                                |
+| iterationsCompleted     | integer | The amount of iterations completed.  |
+| vusActive            | integer  | The amount of VUs active.                                                |
+| vusInitialized           | integer | The amount of VUs initialized from the start of the test.                                                                           |
+| currentTestRunDuration            | float   | The time passed from the start of the current test run in milliseconds.                    |
+
+### VU
+
+| Field               | Type    | Description                                                              |
+|---------------------|---------|--------------------------------------------------------------------------|
+| iterationInInstance | integer | The identifier of the iteration in the current instance.                                                                 |
+| iterationInScenario | integer | The identifier of the iteration in the current scenario.                  |
+| idInInstance        | integer | The identifier of the VU across the instance.                            |
+| idInTest            | integer | The global-unique identifier of the VU.                                  |
```

Comment:
Maybe we should have a general note somewhere that these identifiers are sequential integers starting from 0 and 1 respectfully and that there could be "holes" in them in some cases but in general they will just increase.